lib/insights_cloud.rb

```diff
@@ -1,4 +1,13 @@
 module InsightsCloud
+  module WebUi
+    ADVISOR = 'advisor'
+    VULNERABILITY = 'vulnerability'
+    PATCH = 'patch'
+
+    def self.system_url(subsystem, host_uuid)
+      InsightsCloud.base_url + "/insights/#{subsystem}/systems/#{host_uuid}"
+    end
+  end
```

Comment:
note to myself: move this to lib/foreman_rh_cloud in #276 https://github.com/theforeman/foreman_rh_cloud/pull/276/files#diff-c393d92a34a76151110b7c669c49cb46R3